modules/monitoring/templates/govuk-uptime-collector.conf.erb

```diff
@@ -0,0 +1,5 @@
+description "govuk-uptime-collector"
+
+start on runlevel [2345]
+
+exec /usr/lib/rbenv/versions/2.3.1/bin/ruby /usr/local/bin/govuk_uptime_collector <%= @environment %> publishing-api content-store specialist-publisher travel-advice-publisher publisher whitehall
```

Comment:
Is it normal for us to tie to a particular ruby version like this? I just wonder that this might get missed if ruby 2.3 support is dropped. I wonder if there's a way we can execute rbenv and just tell it latest version of ruby 2?